### PR TITLE
Feat/BackDrop-and-basic-committee

### PR DESCRIPTION
**Pull request by JohanHjelsethStorstad**
This PR adds
- Committee table (must be changed to fit with group system)
- BackDrop-image component
- Some structure for committee pages including /committees, /committees/[name] and /committees/[name]/admin, non of these really have any other purpose than testing for now.
- Upload image has been refactored to its own component for reuse.
- ImageList now has functionality to upload an image to the collection displaying in ImagePagingContext (this is used in ImageList for creating a committee).
- Ability to read and create committees (probably needs changing due to group system)

### CODE REVIEW

File: src/app/admin/committees/ChooseCommitteeLogo.tsx
Comment (theodorklauritzen) [inline src/app/admin/committees/ChooseCommitteeLogo.tsx:1]: Why is this empty

**Review by theodorklauritzen — APPROVED** [review]
LGTM

File: src/actions/groups/committees/read.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import prisma from '@/prisma'
+import type { ExpandedCommittee } from './Types'
+import type { ActionReturn } from '@/actions/Types'
+
+/**
+ * Reads all committees
+ */
+export async function readCommitees(): Promise<ActionReturn<ExpandedCommittee[]>> {
+    // TODO: This should be protected by a permission
+
```
Comment (AndreasVJ) [inline src/actions/groups/committees/read.ts:12]: Typo

File: src/app/committees/[name]/getCommittee.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import type { PropTypes } from './page'
+
+/**
+ * A function to get a committee from params in a page under /committees/[name]
+ * @param params - The name in an object resived by the page
+ * @returns
+ */
+export default async function getCommitee(params: PropTypes['params']) {
+    const name = decodeURIComponent(params.name)
+    const res = await readCommitee(name)
```
Comment (AndreasVJ) [inline src/app/committees/[name]/getCommittee.ts:12]: Typo

File: src/app/committees/[name]/admin/page.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import getCommitee from '@/app/committees/[name]/getCommittee'
+import CmsImage from '@/app/components/Cms/CmsImage/CmsImage'
+import type { PropTypes } from '@/app/committees/[name]/page'
+
+
+export default async function ComitteeAdmin({ params }: PropTypes) {
+    const committee = await getCommitee(params)
+    return (
```
Comment (AndreasVJ) [inline src/app/committees/[name]/admin/page.tsx:8]: Typo

**Review by AndreasVJ — APPROVED** [review]
Looks good

**Pull request merged**